packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/compute-order/DatacentreComputeOrder.page.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import {
+  Datagrid,
+  DatagridColumn,
+  Description,
+  ErrorBanner,
+  Subtitle,
+} from '@ovh-ux/manager-react-components';
+import { OsdsButton } from '@ovhcloud/ods-components/react';
+import { ODS_BUTTON_SIZE, ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  COMPUTE_ORDER_MAX_QUANTITY,
+  COMPUTE_ORDER_MIN_QUANTITY,
+} from './DatacentreComputeOrder.constants';
+import {
+  ComputeOrderVhostCell,
+  ComputeOrderCpuSpeedCell,
+  ComputeOrderRamCell,
+  ComputeOrderCpuCountCell,
+  ComputeOrderPriceCell,
+  ComputeOrderSelectCell,
+} from '@/components/datagrid/compute/ComputeOrderCells.component';
+import Loading from '@/components/loading/Loading.component';
+import { QuantitySelector } from '@/components/datagrid/compute/QuantitySelector.component';
+import { subRoutes } from '@/routes/routes.constant';
+import { IVdcOrderableVhostPriced } from '@/types/vcd-vdc-orderable-resource.interface';
+import { validateComputeQuantity } from '@/utils/formValidation';
+import { useVdcOrderableResource } from '@/data/hooks/useOrderableResource';
+import { useVcdCatalog } from '@/data/hooks/useVcdCatalog';
+import { getPricedOrderableVhostList } from '@/utils/getPricedOrderableResource';
+import useVcdOrder from '@/data/hooks/useVcdOrder';
+
+export default function ComputeOrderPage() {
+  const { t } = useTranslation('hpc-vmware-managed-vcd/datacentres/compute');
+  const { id, vdcId } = useParams();
+  const navigate = useNavigate();
+  const [selectedVhost, setSelectedVhost] = useState<string>(null);
+  const [selectedQuantity, setSelectedQuantity] = useState<number>(1);
+  const { data, isLoading, isError } = useVdcOrderableResource(id, vdcId);
+  const {
+    data: catalog,
+    isLoading: isLoadingCatalog,
+    isError: isCatalogError,
+  } = useVcdCatalog(id);
```

Comment:
```suggestion
  const { data: vcdOrder, isLoading: isLoadingVcdOrder, isError: isVcdOrderError } = useVdcOrderableResource(id, vdcId);
  const {
    data: catalog,
    isLoading: isLoadingCatalog,
    isError: isCatalogError,
  } = useVcdCatalog(id);
```

Maybe rename the return values to be more specific ?